.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
         es2021: true,
     },
     rules: {
+        "comma-dangle": ["error", "only-multiline"],
```

Comment:
Should this not be `always-multiline`?

> "always-multiline" requires trailing commas when the last element or property is in a different line than the closing ] or } and disallows trailing commas when the last element or property is on the same line as the closing ] or }

Reply:
With reflection, this is probably the right move. I've changed to `always-multiline` - let's merge, and then I'll raise a separate PR fixing all of the issues on both `dev` and `4.0.0` (I think the merge from `dev` to `4.0.0` will be messy)? Seems to be a slightly bigger problem than we think, there is a lot of disparity.